src/UserSpaceInstrumentation/Trampoline.h

```diff
@@ -56,6 +58,35 @@ namespace orbit_user_space_instrumentation {
                                                                     const AddressRange& code_range,
                                                                     uint64_t size);
 
+// Returns the set of the instruction pointers from all the threads of a halted tracee.
+[[nodiscard]] ErrorMessageOr<absl::flat_hash_set<uint64_t>> AllInstructionPointersFromProcess(
+    pid_t pid);
+
+// When overriding instructions at the beginning of a function we need to know the number of bytes
+// taken up by the instructions we hit with the write operation. Example:
+//
+// Beginning of the code we want to override:
+// 0x5583ba2837a0: push         rbp
+// 0x5583ba2837a1: mov          rbp, rsp
+// 0x5583ba2837a4: sub          rsp, 0x10
+// 0x5583ba2837a8: mov          rax, qword ptr fs:[0x28]
+//
+// We'd like to override from 0x5583ba2837a0 with a jmp which is five bytes long:
+// 0x5583ba2837a0: jmp          0x5583ba0bf000
+// 0x5583ba2837a5: XYZ
```

Comment:
nop?

Reply:
I don't get this. Are you saying I should override this memory with nops in the procduction code? Maybe, but not really needed. 

Or are you asking if these are nops? Not at all, it's just garbage as described below. These are the last three bytes of the encoding of `sub rsp, 0x10`.

Reply:
Tried to clarify this in the example.